recipe/build.sh

```diff
@@ -15,4 +15,4 @@ EOF
 chmod +x test/suites/api/check-exports
 
 make install
-make check || { cat "${SRC_DIR}/test/test-suite.log"; exit 1; }
```

Comment:
Would keep this here

Reply:
@jakirkham `make check` invokes the test suite. This is failing on non-native environment (e.g. which seems like it should live in `test`?

Reply:
Looking at `libuv` they took a conditional approach

https://github.com/conda-forge/libuv-feedstock/blob/main/recipe/build.sh#LL40-L42C3

Reply:
Yeah it is difficult to separate `make check` specifically from the build environment. So maintainers typically just handle this in the build with conditional checks (as noted)